ui/page/info/info_page.go

```diff
@@ -422,7 +422,7 @@ func (pg *WalletInfo) listenForNotifications() {
 	}
 
 	txAndBlockNotificationListener := &sharedW.TxAndBlockNotificationListener{
-		OnTransaction: func(transaction *sharedW.Transaction) {
+		OnTransaction: func(walletID int, transaction *sharedW.Transaction) {
```

Comment:
why do we need to pass the wallet id here

Reply:
Saw a todo that required ignoring a tx if the wallet selected is different from the wallet that owns the tx.

Reply:
Please see: https://github.com/crypto-power/cryptopower/pull/313/files#r1420727718